Python/codegen.c

```diff
@@ -2546,7 +2547,8 @@ codegen_try_star_except(compiler *c, stmt_ty s)
     for (Py_ssize_t i = 0; i < n; i++) {
         excepthandler_ty handler = (excepthandler_ty)asdl_seq_GET(
             s->v.TryStar.handlers, i);
-        location loc = LOC(handler);
+        /* Set location to the entire line of the except keyword */
+        location loc = LOCATION(handler->lineno, handler->lineno, 0, 0);
```

Comment:
My suggestion here:
```suggestion
        location loc = LOCATION(handler->lineno, handler->lineno, handler->col_offset, handler->col_offset + 6); // len("except")
```
This highlights the `except` keyword.

For plain `except` you could do the same, or dig the coordinates out of the type expression (a bit harder because there are a lot of possibilities there).